src/mount.ts

```diff
@@ -413,16 +413,33 @@ export function mount(
   // users expect stubs to work with globally registered
   // components, too, such as <router-link> and <router-view>
   // so we register those globally.
-  // https://github.com/vuejs/vue-test-utils-next/issues/249
+  // ref: https://github.com/vuejs/vue-test-utils-next/issues/249
+  // we register both the hyphenated version and non-hyphenated
+  // version for good measure.
+  // ref: https://github.com/vuejs/vue-test-utils-next/issues/425
   if (global?.stubs) {
     for (const [name, stub] of Object.entries(global.stubs)) {
       const tag = hyphenate(name)
```

Comment:
IIRC, when a component is registered in PascalCase, it works in the template regardless of whether the template uses kebab-case or PascalCase. maybe we could solve this without double registration by replacing this with a `toPascalCase()`?

Reply:
Hmm neat idea, I will give this a try